### PR TITLE
Fix ag-grid theme enterprise stories

### PR DESCRIPTION
**Pull request by ZarkoPernar**

### CODE REVIEW

File: packages/ag-grid-theme/stories/ag-grid.doc.stories.mdx [hunk 1]
```diff
@@ -125,6 +125,19 @@ Use the 'Salt' or 'UITK' button in each example below to toggle between _Salt AG
   <Story id="data-grid-ag-grid-theme--basic-grid" />
 </Canvas>
 
+#### Variants
+
+There are three variants of the Salt ag-grid theme: `Primary` (default), `Secondary`, and `Zebra`. To use one of the non-default variants add an additional className to the contianer element above `<AgGridReact/>`.
+
+The classNames are:
```
Comment (joshwooding) [inline packages/ag-grid-theme/stories/ag-grid.doc.stories.mdx:132]:
```suggestion
The class names are:
```

Comment (joshwooding) [reply packages/ag-grid-theme/stories/ag-grid.doc.stories.mdx:132]: We should update the references above too

**Review by feliciackh — APPROVED** [review]
LGTM

**Comment by origami-z** [comment]
The invalid license key ag grid watermark is probably not intended.

**Pull request merged**